docs/configuration.md

```diff
@@ -279,9 +279,17 @@ In the runtime the image names are calculated as `my.own.repo/calico/kube-contro
 
 ### `spec.extensions.storage`
 
-`spec.extensions.storage` controls bundled storage.
+`spec.extensions.storage` controls bundled storage provider.
 The default value `external` makes no storage deployed.
-Value `openebs_local_storage` deploys OpenEBS with the local path set up.
+
+To enable [embedded host-local storage provider](storage.md#bundled-openebs-storage) use the following configuration:
```

Comment:
I'd use the same wording as above.
embedded host-local storage provider
->
OpenEBS host-local storage provider
(now there's "embedded" here and "bundled" in the link)